.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+labels: feature
+
+---
+
+<!-- 🚨 STOP 🚨 STOP 🚨 STOP 🚨
+
+LoopBack version 3 is in LTS mode, we are not accepting new features.
```

Comment:
I'm thinking whether we should include the "feature request" template because we won't be accepting new features.  But this might already be a better place to reiterate to our users.  
One suggestion... would it make sense to mention LB4 is in active development, and encourage them to upgrade and submit feature request over there?

Reply:
If we include the "feature" template, then if they decide to open a feature request, then it will be correctly labelled as "feature" and thus it will be easier for us to quickly find such issues and close them.

I am concerned that if there is no template for features, users will open a free-form issue which will make it more difficult for us to find such requests.

I like the idea of mentioning LB4, I'll update the template :+1: